### ISSUE TITLE
[Greetings] Appreciation :partying_face: 

### ISSUE THREAD

**Issue by uncomfyhalomacro** [alstat/Bagyo.jl#4]
Hello! Just found out about you and all your projects :eyes: . I was looking for other Filipinos that use Julia since there is lil to no one I know of. I am glad that I found out about you :tada: 

Hoping for more Filipinos to use Julia :smile:

**Comment by alstat** [comment]
Thank you very much, @uncomfyhalomacro! I looked into your profile as well when you started following me, and I am happy to know that you've seen the beauty of Julia as well. You are the first Filipino I know to have used Julia. 🥳  In my team of Data Scientist, most of them are either user of Python or R. Julia was first introduced to me in 2016 by a French instructor in one of my workshop in WorldBank who suggest to translate the Python code he made to Julia, since according to him it has a lot of potential, and from there I used Julia in 2017 for my Master's Thesis, and the rest is history.

Maraming Salamat, Soc! 😊
